### PR TITLE
Remove polyfills from loaders and add test

### PR DESCRIPTION
**Pull request by metal-messiah**
### Overview
This PR removes `core-js` polyfills from top of CDN loader files.  Polyfills are still generated as standalone file.

### Testing
A test has been added that runs against sub-es6 browsers with the standalone polyfill added as a script above the loader.  This test just checks that the rum call is made.

[Test Run](https://github.com/newrelic/newrelic-browser-agent/actions/runs/3155825666)

[Tests](https://github.com/newrelic/newrelic-browser-agent/actions/workflows/tests.yml)

[Polyfill Tests](https://github.com/newrelic/newrelic-browser-agent/actions/workflows/tests-polyfill.yml)

### CODE REVIEW

File: tools/jil/package.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
     "minimatch": "^3.0.3",
     "multiparty": "^4.1.2",
     "newrelic": "^4.11.0",
+    "node-fetch": "^3.2.10",
```
Comment (cwli24) [inline tools/jil/package.json:26]: FYI node v17.5+ introduced fetch (dev-dependencies) natively

Comment (metal-messiah) [reply tools/jil/package.json:26]: Nice!  that's a good question.  We'd have to bump a few things, like our GH actions as well, which i think run in v14.x

**Comment by metal-messiah** [comment]
[Tests](https://github.com/newrelic/newrelic-browser-agent/actions/workflows/tests.yml)

**Review by cwli24 — APPROVED** [review]
Nice stuff. That 'stn' in testing catch though... heh pain.

**Pull request merged**